### PR TITLE
Docs on client

### PR DESCRIPTION
**Pull request by kijk2869**
Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

### CODE REVIEW

**Review by sannoob — APPROVED** [review]

File: docs/client/client.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# ClientManager
+
+## ``class discodo.ClientManager(**kwargs)``
+
+This class manage voice client
```
Comment (wonderlandpark) [inline docs/client/client.md:5]:
```suggestion
This class represents voice client
```

**Review by wonderlandpark — CHANGES_REQUESTED** [review]

File: docs/client/client.md (hunk 1 above)
Comment (kijk2869) [reply docs/client/client.md:5]: `manage` is correct.

`ClientManager` don't represent voice client.

Comment (wonderlandpark) [inline docs/client/client.md:5]:
```suggestion
A manager of voice client
```

**Review by wonderlandpark — CHANGES_REQUESTED** [review]

File: docs/client/client.md (hunk 1 above)
Comment (wonderlandpark) [reply docs/client/client.md:5]: Better Grammar

**Comment by kijk2869** [comment]
Summon @SaidBySolo

**Comment by SaidBySolo** [comment]
?

**Comment by SaidBySolo** [comment]
@sannoob

**Pull request closed without merging**